en-US/M.xml

```diff
@@ -219,6 +219,27 @@
 
 			</listitem>
 
+		</varlistentry>
+		 <varlistentry id="mission-critical">
+			<term>mission-critical</term>
+			 <listitem>
+				<para>
+					<emphasis>adj.</emphasis> Write hyphenated. 
```

Comment:
Can we standardise on how we write this? E.g., in the style guide right now we tend to use "Write as shown" although I have also seen "Hyphenate as a compound adjective". "Write hyphenated" doesn't really do it for me.